packages/database/scripts/lint_schemas.ts

```diff
@@ -0,0 +1,28 @@
+import { exec } from "node:child_process";
```

Comment:
let's try to stick with camelCase to be consistent with the other files in this folder